examples/10_mono_depth_mobilenetssd.py

```diff
@@ -109,16 +101,51 @@ def displayFrame(name, frame):
             detections = inDet.detections
 
         if inDepth is not None:
-            depthFrame = inDepth.getFrame()
-            depthFrame = cv2.normalize(depthFrame, None, 255, 0, cv2.NORM_INF, cv2.CV_8UC1)
-            depthFrame = cv2.equalizeHist(depthFrame)
-            depthFrame = cv2.applyColorMap(depthFrame, cv2.COLORMAP_HOT)
+            depthFrame = cv2.flip(inDepth.getFrame(), 1)
+            # frame is transformed, the color map will be applied to highlight the depth info
+            depthFrame = cv2.applyColorMap(depthFrame, cv2.COLORMAP_JET)
+
+            # Uncomment one of these and comment the one given above
+            # to see visualisation in different color frames
+
+            # depthFrame = cv2.applyColorMap(depthFrame, cv2.COLORMAP_BONE)
```

Comment:
is this really needed? I think a few color maps would suffice and maybe a link to cv2 docs where all others are listed. Same goes for the `03_depth_preview.py`

Reply:
I think that these encourage you to actually try different ones, and you don't need to put anything together yourself. This was added in https://github.com/luxonis/depthai-python/pull/127 so I thought we want to have it here too